README.md

```diff
@@ -1,197 +1,45 @@
-
-![](docs/img/release-page-cli.png)
-
-For example :
-```
-./cli --db-type mongo --mongo-uri mongodb://localhost:27017/sample update-db --path $script_path --identifier-regex "(.*)"
-```
-
-This command will start Datamaintain on a mongo db, the mongo is accessible with URI `mongodb://localhost:27017/sample`.
-The folder path containing scripts is `$script_path`. 
-
-### Use docker
-You can use the CLI via a docker image. You just need to mount the script path to the container :
-```
-docker run --rm --volume $script_path:/scripts docker.pkg.github.com/4sh/datamaintain/datamaintain:1.2-mongo-4.4 --db-type mongo --mongo-uri mongodb://localhost:27017/sample update-db --path /scripts --identifier-regex "(.*)"
-```
-
-In this example :
-* `$script_path` is a path to the script folder
-* `--rm` will remove the container once Datamaintain is finish
-* `--volume` mounts the script folder on your machine to the Datamaintain container with the path `/scripts`
-* After the image name `datamaintain` you can pass arguments to the cli normally.
-* On Mac OS you may need to replace `localhost` in the mongo URI 
-  by `host.docker.internal`. See [docker documentation](https://docs.docker.com/docker-for-mac/networking/).
-
-Datamaintain image use a mongo shell.
-
-Image tag has form `<datamaintain version>-<db type>-<db version>` for example 
-`docker.pkg.github.com/4sh/datamaintain/datamaintain:1.2-mongo-4.4` is a datamaintain 1.2 with a mongo shell 4.4.
-For now, datamaintain only support mongo database.
-You can see all images [here](https://github.com/orgs/4sh/packages?repo_name=datamaintain)
-
-
-## Installation in a project with already executed scripts
-
-When you already have executed scripts on your project and you want to start using Datamaintain, please follow those steps:
-- Add Datamaintain as a dependency to your project, as described [here](README.md#installation). 
-- Download the CLI from the version you are aiming for. The CLI is released as an asset in every Datamaintain release, you may find it in the [releases](https://github.com/4sh/datamaintain/releases).
-- Execute the CLI using the following command replacing the arguments with the values you want. An explanation about each configuration key is provided here.
-```bash
-
-./datamaintain-cli --db-type $DB_TYPE --mongo-uri $MONGO_URI update-db --path $PATH --identifier-regex $REGEX --execution-mode FORCE_MARK_AS_EXECUTED
-```
-
-## Executed scripts in your database
-Your scripts executions will be stored in your database. In Mongo, you will have a collection named ```executedScripts``` that will contain executed scripts, as defined below:
-
-| Attribute | Description |
-|---|---|
-| id | | 
-| name | | 
-| checksum | | 
-| identifier | | 
-| executionStatus | | 
-| executionDurationInMillis | Duration of your script execution, in milliseconds. | 
-| executionOutput | | 
+### Overview of the features
+- [The execution reports](docs/executedscripts.md)
+- The tag feature (work in progress),
```

Comment:
Tag feature is done.
But the main information of that is to have possibility to whitelist or blacklist some scripts (via tags)